psi4/src/psi4/libpsio/filemanager.cc

```diff
@@ -63,8 +63,22 @@ PSIOManager::~PSIOManager() {}
 
 std::shared_ptr<PSIOManager> PSIOManager::shared_object() { return _default_psio_manager_; }
 
-void PSIOManager::set_default_path(const std::string& path) { default_path_ = path + "/"; }
-void PSIOManager::set_specific_path(int fileno, const std::string& path) { specific_paths_[fileno] = path + "/"; }
+void PSIOManager::set_default_path(const std::string& path) {
+    if (path.size() && (path.back() == '/')) {
```

Comment:
It's a little cleaner to check `!path.empty()` for the first condition.

Reply:
done locally. I'll wait for more reviews before pushing.